### ISSUE TITLE
set up feeder in a vps?

### ISSUE THREAD

**Issue by xxxsomethingxd** [davidmann4/ogar-feeder-bot#342]
can i setup the feeder on a vps and log in from my pc only from the browser and get bots?

**Comment by hassanila** [comment]
YES

**Comment by xxxsomethingxd** [comment]
@hassanila97 can you help me on skype please

**Comment by hassanila** [comment]
What OS on vps?

**Comment by xxxsomethingxd** [comment]
windows amazon ec2

**Comment by hassanila** [comment]
skype: hassanila97@hotmail.com

**Comment by xxxsomethingxd** [comment]
i added u my id is the-joker906 can u accept?

**Comment by skyleter** [comment]
I was about to help you but isnt ubuntu or centos :(

**Comment by picusc** [comment]
Ok, assume it is ubuntu... What to do then ?

**Comment by philw1508** [comment]
Can some help me with a windows vps to set it up for bots?

**Comment by Qasimmodo** [comment]
Yes. edit config.js and set the server to ws://bots.myservers.io:8081 then, locate the userscript and edit the userscript and also set it to ws://bots.myservers.io:8081 . You can use "find" (CTRL + F) to look for it, just search for ws:// or 8081 and you'll probably find it. Then simply put your UUID in config.js. Then run the server-sided script.

Sorry if that was confusing, hope I helped.
